flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliClient.java

```diff
@@ -532,6 +535,14 @@ private void callShowFunctions() {
 				.collect(Collectors.toList());
 	}
 
+	private List<String> getShowResult(String objectToShow, SqlCommandCall cmdCall) {
+		TableResult tableResult = executor.executeSql(sessionId, "SHOW " + objectToShow + " " + cmdCall.operands[0]);
```

Comment:
I think `cmdCall.operands[0]` contains the whole SQL statement, no?

Reply:
Yeah, I check the `parseBySqlParser` method, and confirm the `cmdCall.operands[0]` contains the whole SQL statement.